src/invidious/views/components/item.ecr

```diff
@@ -96,6 +96,7 @@
                 </div>
                 <% end %>
             </div>
+        <% when Category %>
```

Comment:
I iagine that this is "TODO"?

Reply:
It's more of a placeholder. The range of designs the YT categories has is far too varied for us to have it as a component.

Reply:
Ah, okay